src/js/providers/html5.js

```diff
@@ -508,20 +495,29 @@ define([
             _videotag.muted = !!state;
         };
 
-        function _sendBufferUpdate() {
-            if (!_attached) { return; }
-            var newBuffer = _getBuffer();
-
-            if (newBuffer >= 1) {
-                clearInterval(_bufferInterval);
+        function _checkBufferAndPlayback() {
+            if (!_attached) {
+                return;
             }
 
-            if (newBuffer !== _bufferPercent) {
-                _bufferPercent = newBuffer;
+            var buffered = _getBuffer();
+            if (buffered !== _buffered && buffered <= 1) {
```

Comment:
Defensive move to insure bufferPercent never exceeds 100. It maintains the logic that was removed, but does not clear the interval:
```
if (newBuffer >= 1) {
   clearInterval(_bufferInterval);
```